lib/mix/lib/mix/tasks/test.ex

```diff
@@ -411,7 +436,12 @@ defmodule Mix.Tasks.Test do
   end
 
   defp manifest_opts(opts) do
-    Keyword.put(opts, :manifest_path, Mix.Project.manifest_path())
+    manifest =
+      Mix.Project.manifest_path()
+      |> Path.join(".ex_unit_results.elixir")
+      |> ExUnit.Manifest.read()
```

Comment:
I would prefer if we didn't access ExUnit.Manifest in the Mix application. It is private API. I truly prefer the previous approach where we simply gave it a path. :)